restcomm.android.client.sdk/src/main/java/org/mobicents/restcomm/android/sipua/impl/SipManager.java

```diff
@@ -878,6 +882,11 @@ else if (response.getStatusCode() == Response.RINGING) {
 			//RCLogger.i(TAG, "BUSY");
 			dispatchSipEvent(new SipEvent(this,
 					SipEventType.SERVICE_UNAVAILABLE, "", ""));
+		} else if (response.getStatusCode() == Response.FORBIDDEN) {
```

Comment:
@atsakiridis Should the error code here change as well? Something like SIGNALLING_INSTANT_MESSAGE_ERROR for MESSAGE and SIGNALLING_CALL_ERROR for INVITE?